nodewatcher/plugins/torque.py

```diff
@@ -18,7 +18,8 @@
 
 log = logging.getLogger(__name__)
 
-def runPipe(cmds):
+
+def run_pipe(cmds):
```

Comment:
To have a cleaner patch, it is better to split the function name refactoring from the new functionalities.